docs/docs/api-ref.md

```diff
@@ -0,0 +1,261 @@
+---
+title: API reference
+layout: docs
+---
+
+<div class="alert alert-info">
+    <b>Important:</b> More coming soon! This section is under active construction and might not reflect all the available functionality of the TSC library.
+    Until this reference is completed, we have noted the source files in the TSC library where you can get more information for individual endpoints.
+</div>
+
+* TOC
+{:toc}
+
+## Authentication
+
+Source files: server/endpoint/auth_endpoint.py, models/tableau_auth.py
+
+### Sign In
+
+Signs you in to Tableau Server.
+
+```py
+Auth.sign_in(authentication_object)
+```
+
+### Sign Out
+
+Signs you out of Tableau Server.
+
+```py
+Auth.sign_out()
+```
+
+## Sites
+
+Source files: server/endpoint/sites_endpoint.py, models/site_item.py
+
+### Create Site
+
+Creates a new site for the given site item object.
```

Comment:
Big meta-question: Do we want to call them <Noun> Item Objects, or just <Noun> Objects? I vote latter. I know we call them SiteItem in the API but `item object` sounds redundant